package-lock.json

```
@@ -1,12 +1,12 @@
 {
   "name": "rlnjs",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

The version does not match package.json, which is 2.0.7. We probably need an `npm install`
```suggestion
  "version": "2.0.7",
```

---

Good catch!